modules/rosa-architecture.adoc

```diff
@@ -1,21 +1,31 @@
 // Module included in the following assemblies:
 //
 // * rosa_architecture/rosa_architecture_sub/rosa-architecture-models.adoc
-[id="rosa-architecture_{context}"]
-= ROSA architecture on public and private networks
 
-You can install ROSA using either a public or private network. Configure a private cluster and private network connection during or after the cluster creation process.
-Red Hat manages the cluster with limited access through a public network. For more information, see xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-service-definition[ROSA service definition].
+[id="rosa-classic-architecture_{context}"]
+= ROSA Classic architecture
+
+In ROSA Classic, both the control plane and the worker nodes are deployed on the customer's VPC subnets.
+
+[id="rosa-classic-architecture-networks_{context}"]
+== ROSA Classic architecture on public and private networks
+
+With ROSA Classic, you can create your clusters on public or private networks.
+
+Customers can customize access patterns for their API server endpoint and Red Hat SRE management in the following ways:
+* Public - API server endpoint and application routes are internet-facing.
+* Private - API server endpoint and application routes are private. Private ROSA Classic clusters use some public subnets, but no control plane or worker nodes are deployed in public subnets.
+* Private with AWS PrivateLink - API server endpoint and application routes are private. Public subnets or NAT gateways are not required in your VPC for egress. ROSA SRE management uses AWS PrivateLink.
+
+The following image depicts the architecture of both public and private networks.
```

Comment:
The following image depicts the architecture of ROSA Classic cluster deployed with both public and private networks.